packages/core/src/utils/formatPrice.ts

```diff
@@ -31,30 +31,38 @@ export default function formatPrice(
     amount = Math.abs(amount);
   }
 
-  let amountStr = '';
-
-  // Format as a currency
-  if (currency) {
-    amountStr = amount.toLocaleString(locale, {
-      style: 'currency',
-      currency,
-      currencyDisplay: options.display || 'symbol',
-    });
-
-    // Format as a number
-  } else {
-    amountStr = amount.toLocaleString(locale);
-  }
+  try {
+    let amountStr = '';
 
-  // Remove ending zeros
-  if (options.trimTrailingZeros) {
-    amountStr = amountStr.replace(/\.0+$/, '');
-  }
+    // Format as a currency
+    if (currency) {
+      amountStr = amount.toLocaleString(locale, {
+        style: 'currency',
+        currency,
+        currencyDisplay: options.display || 'symbol',
+      });
 
-  // Wrap the value in parenthesis if negative
-  if (negative) {
-    amountStr = `(${amountStr})`;
-  }
+      // Format as a number
+    } else {
+      amountStr = amount.toLocaleString(locale);
+    }
 
-  return amountStr;
+    // Remove ending zeros
+    if (options.trimTrailingZeros) {
+      amountStr = amountStr.replace(/\.0+$/, '');
+    }
+
+    // Wrap the value in parenthesis if negative
+    if (negative) {
+      amountStr = `(${amountStr})`;
+    }
+
+    return amountStr;
+  } catch (error) {
+    // if (__DEV__) {
+    //   console.error(error);
+    // }
+
+    return '';
```

Comment:
How much would break if we updated this to be null?

Reply:
@milesj a good chunk. ran into issues with `null` when doing the datetime stuff. went with `undefined` and then had to do a lot of type updates in our apps to handle that :/

Reply:
I'm just worried that downstream consumers wouldn't be aware that invalid = empty string.

Reply:
good point! yeah, let's change it to `null` then